crates/btcio/src/rpc/types.rs

```diff
@@ -187,6 +189,79 @@ pub struct ScriptPubkey {
     pub address: Option<String>,
 }
 
+/// Result of JSON-RPC method `submitpackage`.
+///
+/// > submitpackage ["rawtx",...] ( maxfeerate maxburnamount )
+/// >
+/// > Submit a package of raw transactions (serialized, hex-encoded) to local node.
+/// > The package will be validated according to consensus and mempool policy rules. If any
+/// > transaction passes, it will be accepted to mempool.
+/// > This RPC is experimental and the interface may be unstable. Refer to doc/policy/packages.md
+/// > for documentation on package policies.
+/// > Warning: successful submission does not mean the transactions will propagate throughout the
+/// > network.
+/// >
+/// > Arguments:
+/// > 1. package          (json array, required) An array of raw transactions.
+/// > The package must solely consist of a child and its parents. None of the parents may depend on
+/// > each other.
+/// > The package must be topologically sorted, with the child being the last element in the array.
+/// > [
+/// > "rawtx",     (string)
+/// > ...
+/// > ]
+#[allow(clippy::doc_lazy_continuation)]
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct SubmitPackage {
+    /// The transaction package result message.
+    ///
+    /// "success" indicates all transactions were accepted into or are already in the mempool.
+    pub package_msg: String,
+    /// Transaction results keyed by wtxid.
+    #[serde(rename = "tx-results")]
+    pub tx_results: BTreeMap<String, SubmitPackageTxResult>,
+    /// List of txids of replaced transactions.
+    #[serde(rename = "replaced-transactions")]
+    pub replaced_transactions: Vec<String>,
+}
+
+/// Models the per-transaction result included in the JSON-RPC method `submitpackage`.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct SubmitPackageTxResult {
+    /// The transaction id.
+    pub txid: String,
+    /// The wtxid of a different transaction with the same txid but different witness found in the
+    /// mempool.
+    ///
+    /// If set, this means the submitted transaction was ignored.
+    #[serde(rename = "other-wtxid")]
+    pub other_wtxid: Option<String>,
+    /// Sigops-adjusted virtual transaction size.
+    pub vsize: Option<i64>,
```

Comment:
Could this be just `i64` or `u64`? The docs does not say it is optional.

> `"vsize" : n,                  (numeric) Virtual transaction size as defined in BIP 141.`

Reply:
Makes sense, done in 53af954a5166b1db43a14c87c5105d4d3b1d7f49